Eastern-Research-Group/mywaterway
# Feature/248 date slider tooltip

## Related Issues:
* [HMW-248](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-248&quickFilter=2479)

## Main Changes:
* Added a hover tooltip next to the date slider heading.
* Updated the `package.json` file to include `@reach/tooltip`.

## Steps To Test:
1. Go to the _Past Water Conditions_ sub-tab on the **Monitoring** tab of the **Community** page
2. Look for the **(?)** icon next to the heading "Date range for the ___ watershed"
3. Hover over the icon to confirm that the tooltip appears
